helpers/sendVerificationEmail.js

```diff
@@ -0,0 +1,23 @@
+import { DOMAIN, client } from "../config/mailgunConfig.js";
+
+export function sendVerificationEmail(email, verificationToken) {
+  const messageData = {
+    from: "emiliakielinska96@gmail.com",
+    to: email,
+    subject: "Email verification link",
+    text: `Hello! Your email verification link: /users/verify/${verificationToken}`,
```

Comment:
Ten link nie zadziała bo powinno być przekierowanie z email do serwera. Potrzebujesz jeszcze domeny protokołu. Czy jak testowałaś tę aplikację to co się pojawiło po kliknięciu w email?

Reply:
Masz rację, dodałam protokół i host do linka. Testowałam wszystko w postmanie, i pewnie dlatego nie zwróciłam na to szczególnej uwagi. :)

Reply:
Poprawiłam jeszcze te funkcje o których pisałeś. Zostawiłam jedynie arrow function w service/index.js ponieważ to są tak krótkie funkcje, że podobają mi się bardziej w takiej postaci :) A cała reszta plików zawiera zwykłą deklarację funkcji. Dzięki, że zwróciłeś na to uwagę :)